### PR TITLE
feat: Move to RKD endpoint

### LINKED ISSUE #720: Improve RKDArtists results
When searching in RKDArtists it is quite difficult to disambiguate between the different results, see the results of searching for 'Vincent van Gogh'. It is hard to identify the famous painter from this list. It would help to add the information from the schema:additionalType to skos:scopeNote. In this case this would at least help to identify the painter(s) in the result list.

### LINKED ISSUE #1128: Add RKD birth and death date to skos:scopeNote

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #1128
Fix #720

### CODE REVIEW

File: packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq [hunk 1]
```diff
@@ -1,27 +1,69 @@
...
-    VALUES ?type { schema:Person schema:Organization } .
+            ?uri ?name ?label .
+            VALUES ?name { schema:name schema:alternateName }
 
-    OPTIONAL { ?uri schema:name ?schema_name . }
-    OPTIONAL { ?uri schema:alternateName ?schema_alternateName . }
-    OPTIONAL { ?uri schema:description ?schema_description . }
+            OPTIONAL { ?uri schema:description ?schema_description . }
+            OPTIONAL { ?uri schema:birthPlace ?birthPlace .
+                FILTER(langMatches(lang(?birthPlace),"en")) }
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq:33]: @RubenSchalk Is the `en` intentional? Shouldn’t we prefer Dutch information by default?

Comment (rschalkrce) [reply packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq:33]: @ddeboer sdo:Description zit er enkel als @en in zie ik, dus dat zal de overweging zijn geweest om alle tags in @en te doen (vermoed ik). Laten we het maandag meenemen in ons overleg over meertaligheid en hoe daarom mee om te gaan.

File: packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq [hunk 2]
```diff
@@ -1,28 +1,68 @@
...
+        skos:label ?label .
 }
 WHERE {
-    ?uri a ?type .
-    VALUES ?type { schema:Person schema:Organization }
-    ?uri ?name ?label .
-    VALUES ?name { schema:name schema:alternateName }
-
     ?uri ?predicate ?label .
     ?label <bif:contains> ?virtuosoQuery .
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq:15]: I’m not sure that we should search _all_ fields. If you search for ‘Vincent Van Gogh’ you’ll find many not so relevant results that mention Van Gogh in their scopeNote. Should we only include schema:name and schema:alternateName?

Comment (rschalkrce) [reply packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq:15]: I was also wondering if we should keep this in. I will check with RKD

File: packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq (hunk 1 above)
Comment (ddeboer) [reply packages/network-of-terms-catalog/catalog/queries/lookup/rkdartists.rq]: `schema:Description` is indeed only available in English, but that one is isn’t filtered in the query.

The other fields, `schema:description`, `:birthplace`, and `:deathPlace` _are_ filtered but available in Dutch too, so I switched to `"nl"`.

File: packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq (hunk 2 above)
Comment (ddeboer) [reply packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq:14]: I limited this for now to `schema:name` and `schema:alternateName` for parity with the old endpoint. If needed, we can revert that later.

**Pull request merged**